html-skeleton/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE HTML>
+<html>
+
+<head></head>
+<title>HTML Skeletons</title>
```

Comment:
The `title` element needs to be a child of the `head` element

Reply:
Hi @codingcodymiller I have made the change. Thanks!